waltz-data/src/main/ddl/liquibase/db.changelog-1.7.xml

```diff
@@ -299,6 +300,20 @@
         </sql>
     </changeSet>
 
+    <changeSet id="20171024-2623-2"
+               author="kamransaleem"
+               dbms="postgresql">
+        <sql>
+            UPDATE data_type
+            SET unknown = TRUE
```

Comment:
Can't we use `unknown = {true.value}` here and just modify the original changeset to include `postresql` as well?

Reply:
I was getting odd digest failures for some `true.value` substitutions. Think this was one of them. No idea why some don't seem to affect the digest and some do...